### DISCUSSION TITLE
丝之歌首发登录电玩店！

### DISCUSSION THREAD

**Discussion by xioagong** [share121/inter-knot#1359] [其他]
![c761074a26b5f107ca0636a21a63cda](https://github.com/user-attachments/assets/11f36c79-f8c9-40ac-9d7c-905b844635c6)
丝之歌将于2024年8月12号于六分街电玩店开售，首发价为114514丁尼，前69位客户可以得到店长的私照（懂得都懂），其他渠道的发售时间还未公布。

**Comment by EhuanyingK** [comment]
我真幸运，刚通关空洞骑士就能玩丝之歌了

**Comment by kallen1314** [comment]
什么？登录电玩店了？店长肯定有关系！不然这种只有空洞消失才会出的游戏怎么可能现在就能玩到

**Comment by ZhuYuan-148** [comment]
嗯…下午要不邀请店长一起去试试。

**Comment by warmgoose** [comment]
我真幸运。。。。后面忘了

**Comment by UmayKui3678** [comment]
建议去b站看一下这个BV1GJ411x7h7

**Comment by BEILUO126** [comment]
我真幸运
